scripts/preloadHTTPSE.js

```diff
@@ -1,34 +1,50 @@
 'use strict'
 const fs = require('fs')
-const http = require('https')
+const zlib = require('zlib')
+const path = require('path')
+const https = require('https')
 const levelup = require('level')
 const rmDir = require('./util').rmDir
 const exec = require('child_process').exec
 
-const xpiVersion = '2018.9.19' // Manually update this to latest version
-
 const downloadRulesets = (dir, cb) => {
-  const downloadURL = `https://www.eff.org/files/https-everywhere-${xpiVersion}-eff.xpi`
-  const xpiFile = fs.createWriteStream('httpse.xpi')
-  http.get(downloadURL, (response) => {
-    response.pipe(xpiFile)
-    xpiFile.on('finish', () => {
-      xpiFile.close(() => {
-        exec('unzip ../httpse.xpi', {
-          cwd: 'https-everywhere'
-        }, (err) => {
-          if (err) {
-            throw err
-          } else {
+  let timestamp = ''
+  let baseURL = 'https://www.https-rulesets.org/v1/'
```

Comment:
does EFF guarantee that the rulesets published under `v1` will be compatible with HTTPSE`'2018.9.19'` ? i assume so but just checking because they've done a few updates in the past where the format has changed (and so auto-updating would have broken Brave). 

otherwise this approach looks great

Reply:
The `v1` directory was added as per https://github.com/EFForg/https-everywhere/issues/14907#issuecomment-375744558, so I assume there shouldn't be breaking changes in the rulesets format. Maybe @Hainish @zoracon from EFF can help guarantee that?